frontend/src/metabase/collections/components/BulkActions.tsx

```diff
@@ -68,35 +87,42 @@ function BulkActions({
                   medium
                   purple
                   disabled={!canArchive}
-                  onClick={onArchive}
+                  onClick={() => {
+                    onArchive?.();
+                  }}
                 >{t`Archive`}</CardButton>
               </CardSide>
             </ToastCard>
           </BulkActionsToast>
         )}
       </Transition>
-      {!_.isEmpty(selectedItems) && selectedAction === "copy" && (
-        <Modal onClose={onCloseModal}>
-          <CollectionCopyEntityModal
-            entityObject={selectedItems[0]}
+      {!!selectedItems &&
+        !_.isEmpty(selectedItems) &&
```

Comment:
we do this check twice (and I think I'm going to add 2 more today), could you please move this check to a common named variable.